ups.sh

```diff
@@ -7,13 +7,31 @@ touch $LOG_FILE
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-ARGUMENTS=""
+ARGUMENTS=$(for f in $SCRIPT_DIR/package-finders/*.sh
+	    do
+		$f
+	    done)
 
-export DRY_RUN=true
 echo "
 
 $(date -Iseconds) New run of ups.sh" >> $LOG_FILE
 
+if [ -z "${NIXPKGS+}" ]
+then
+    NIXPKGS=$HOME/Projects/nixpkgs
```

Comment:
This line concerns me because I don't want to guess where someone's nixpkgs repository is because the up.sh script does a bunch of resetting.

Reply:
Yep I can remove it. Does it make sense to just clone a clean NIxpkgs repo each time? That can be a little bit slow sometimes but in the grand scheme of things it shouldn't be significant.